.github/workflows/ci.yaml

```
@@ -45,10 +45,11 @@ jobs:
           - '3.9'
           - '3.10'
           - '3.11'
+          - '3.12'
         biopython-version:
           # list of Biopython versions with support for a new Python version
           # from https://github.com/biopython/biopython/blob/master/NEWS.rst
-          - '1.80' # first to support Python 3.10 and 3.11
+          - '1.82' # first to support Python 3.12
```

```suggestion
          - '1.80' # first to support Python 3.10 and 3.11
          - '1.82' # first to support Python 3.12
```

The new version should be added, not replaced. Then some combinations of biopython+python versions might need to be excluded. See the previous usage:

https://github.com/nextstrain/augur/blob/509c193ac5e8dd008011e927520f138ee019af52/.github/workflows/ci.yaml#L27-L42